extensions/Worker.Extensions.Http.AspNetCore/src/Coordinator/ContextReference.cs

```diff
@@ -59,9 +46,10 @@ internal void CompleteFunction()
 
             if (_httpContextValueSource.Task.IsCompleted)
             {
-                if (_httpContextValueSource.Task.IsCanceled || _token.IsCancellationRequested)
+                if (_httpContextValueSource.Task.IsCanceled || _invocationCancellationToken.IsCancellationRequested)
```

Comment:
We can remove `_httpContextValueSource.Task.IsCanceled`, as we won't be setting this to canceled anywhere.